tools/cpplint

```diff
@@ -1,4 +1,3 @@
 #!/bin/sh
 
-python tools/cpplint.py --filter=-build/include_order,-whitespace/comments --extensions=h,cc ./**/*.h ./**/*.cc
-
+python tools/cpplint.py --filter=-build/include_order,-whitespace/comments --extensions=h,cc $(git ls-files $(git rev-parse --show-toplevel) | grep -E "cppclient/.*[cc|h]")
```

Comment:
The stuff in `tools` should not depend on the files having been added to the index/